sched/scheduler/scheduler_test.go

```diff
@@ -2,6 +2,8 @@ package scheduler
 
 import (
 	"fmt"
+	"testing"
```

Comment:
It seems like we're pushing towards an import style.

Do we have a (living) doc of our coding style? Maybe in our repo?

I'd even be fine having one more level, so it's like:

<stdlib imports>

<3rdparty imports>

<scoot imports>